network-api/networkapi/templates/pages/buyersguide/base.html

```diff
@@ -62,7 +62,7 @@
 {% endblock %}
 
 
-{% block bodyclass %}pni{% endblock %}
+{% block bodyclass %}pni tw-pt-[150px]{% endblock %}
```

Comment:
Could you add a comment explaining why we need a 150px padding here?

Reply:
Mentioned it in the description but since we are using position:fixed for the nav, everything on the body shifted up since the nav is around 150px so I thought adding that height as padding should be enough to make the rest of content look normal. This was the solution I thought about that didn't include using JS to calc the padding just in case people disable JS.